rules/test.bzl

```diff
@@ -160,3 +160,14 @@ def ios_ui_test(name, apple_library = apple_library, **kwargs):
     if not kwargs.get("test_host", None):
         fail("test_host is required for ios_ui_test.")
     _ios_test(name, rules_apple_ios_ui_test, rules_apple_ios_ui_test_suite, apple_library, **kwargs)
+
+def ios_unit_snapshot_test(name, apple_library = apple_library, **kwargs):
```

Comment:
I was thinking if we can just reuse `ios_unit_test` function instead of creating a new one

Reply:
I've got some features conventions for snapshots that I'm hoping to inject here longer term; today we tag a `TestRunner` as as a `snapshot` which can implicate various side effects; ATM it also has to be like this for the lines added in this PR due to other issues but this sets us in the right direction